templates/create_project.py.erb

```diff
@@ -31,25 +32,39 @@ def main():
   if not options.platform:
     parser.error("Platform is required")
 
+  # try to load the requested organization
+  # and the admin user, who will own all new projects and teams
   e = False
   try:  
     o = Organization.objects.get(name=options.org)
-    t = Team.objects.get(name=options.team)
+    u = User.objects.get(email='<%= @admin_email %>')
   except:
     e = sys.exc_info()[0]
-
   if e:
     print "Error loading Sentry environment: %s" % (e)
     sys.exit(1)
 
+  # try to load the requested team
+  try:
+    t = Team.objects.get(name=options.team,organization_id=o.id)
+  except:
```

Comment:
Probably want to be specific on the exception being caught.  Need to test but I believe its `Team.DoesNotExist`.